### DISCUSSION TITLE
Release binary uses strange MITRE SECURITY EVASION (v0.0.36 RC-1)

### DISCUSSION THREAD

**Discussion by selfmadebuba** [thliebig/openEMS-Project#89] [General]
Could someone elaborate why Release binary uses strange MITRE SECURITY EVASION TRICKS (v0.0.36 RC-1)

Mitre ATT&CK Tactics and Techniques:

Defense Evasion:

- Masquerading - Drops files with a non matching file extension (content does not match to file extension)
- Disable or Modify Tools - Creates guard pages, often used to prevent reverse engineering and debugging
- Hidden Files and Directories - Creates hidden files, links and/or directories

Discovery:

- System Information Discovery - Queries the cryptographic machine GUID
- System Information Discovery - Reads software policies
- Security Software Discovery - Uses the "uname" system call to query kernel version information (possible evasion)

https://www.virustotal.com/gui/file/0f81ae95654dd2b6b2fdb9ffa3a21c59cb67a022f12a2c8a2a6f7c6f1f25905d/behavior

**Comment by thliebig** [comment]
Hi, well I always build the windows binaries for openEMS and I'm curious about these different message virustotal comes up with too...
A few month ago virustotal.com supposedly found a trojan inside the openEMS binary...   After a user told me I immediately toke the zip-release offline and scanned my build machine for any kind of virus infestation but found nothing... Building again yielded the same message which left me very confused. But since only a few of many engines used by virustotal had a positive I suspected a false positive? 
In the mean time this message has disappeared and this new one has popped up... If anybody can shed some light on this it would be great.  But since again only a few (this time only one?) engine finds anything it seems again like a false positive?

I'm really not a Windows expert and can everybody only encourage to ditch it to and get a proper operating system ;)
